src/core/internal/util/math.d

```diff
@@ -0,0 +1,53 @@
+// Written in the D programming language
```

Comment:
What's the point of this line?

Reply:
I'm not really sure honestly, but I just followed the criteria around some files in `druntime`/`phobos`.

e.g.:
https://github.com/dlang/druntime/blob/f94e816c1dd229cbb133085a6fc41d1bd6d45594/src/core/math.d#L1